internal/pkg/deploy/cloudformation/stack/testdata/environments/template-with-imported-certs-observability.yml

```diff
@@ -418,6 +421,13 @@ Resources:
     Condition: CreateALB
     Type: AWS::ElasticLoadBalancingV2::LoadBalancer
     Properties:
+      LoadBalancerAttributes:
+        - Key: 'access_logs.s3.enabled'
+          Value: true
+        - Key: 'access_logs.s3.prefix'
+          Value: "accesslogsbucketprefix"
+        - Key: 'access_logs.s3.bucket'
+          Value: "accesslogsbucket"
```

Comment:
I like how it's easier this way to review the changes the code has on the template!